apps/docs/src/guide/tooling/graphql-integration.md

```diff
@@ -0,0 +1,16 @@
+# GraphQL Integration
+
+The Fuel Network provides a GraphQL API to query the blockchain ([docs](https://docs.fuel.network/docs/graphql/overview/)).
+
+For its own purposes, the SDK creates custom operations based off of the API's schema and auto-generates TypeScript client code via codegen tools.
```

Comment:
Should this have a subheading above called Operations?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1979/commits/8d5e7ab97002623311b519a30c1d72e2aefdd9a5